util/format/format_test.go

```diff
@@ -47,6 +48,7 @@ z
 	expect = "abc3%e x y z\n "
 	checkFormat(t, f, buf, str, expect)
 
-	str2 := OutputFormat(`\'\000abc\n\rdef`)
-	assert.Equal(t, str2, "\\''\\000abc\\n\\rdef")
+	str1 := fmt.Sprintf("%c%c%s%c%c%s", '\'', '\000', "abc", '\n', '\r', "def")
+	str2 := OutputFormat(str1)
```

Comment:
the [back quote](https://go.dev/ref/spec#:~:text=character%20sequences%20between-,back%20quotes,-%2C%20as%20in%20%60foo) is a better way to write the case, no need to change this case

Reply:
make check linter and correct code format for show_test.go and commit again, please check. And for the "format_test.go", sure back quote will be better and display right.  but what i confusing is in the old test case, like `\n`, `\r` will be treated as '\' 'r' '\' 'n' seperately, the func "replace" will not be used at all, so if meet a char '\n', will not act like back quote test does.  Anyway, if sure the previous format_test.go testcase is right, so just ignore this file's change

Reply:
get it